src/stories/Library/dropdown/dropdown.scss

```diff
@@ -17,18 +19,22 @@
 }
 
 .dropdown__select {
+  $_padding: 20px;
+
   appearance: none;
   -webkit-appearance: none;
   min-width: 230px;
   width: 100%;
-  height: 50px;
+  height: $_dropdown-icon-size;
   cursor: pointer;
   background-color: transparent;
   color: $color__global-black;
   border-radius: 0;
   border: 1px solid;
   border-color: inherit;
-  padding: 0 50px 0 20px;
+  padding: 0;
+  padding-left: $_padding;
+  padding-right: $_dropdown-icon-size + $_padding;
```

Comment:
As I see it this will change `padding-right` from 50px to 70px (50 + 20). I wonder if that is intentional?

Reply:
It is intentional :) It appears to be broken as it is.